app/models/miq_action.rb

```diff
@@ -310,7 +310,7 @@ def action_email(action, rec, inputs)
       }
     end
 
-    invoke_or_queue(inputs[:synchronous], __method__, "notifier", nil, MiqSnmp, 'queue_email', [email_options])
+    invoke_or_queue(inputs[:synchronous], __method__, "notifier", nil, MiqAction, 'queue_email', [email_options])
```

Comment:
I think I am going to extract this logic into a separate PR.
It is a bug on miq_action.rb that not only affects to this work.